src/types/terminal.go

```diff
@@ -0,0 +1,18 @@
+package types
+
+type StartTerminalUri struct {
+	ClusterName   string `uri:"clusterName" binding:"required"`
+	NamespaceName string `uri:"namespaceName" binding:"required"`
+	PodName       string `uri:"podName" binding:"required"`
+	ContainerName string `uri:"containerName" binding:"required"`
+}
+
+type StartTerminalBody struct {
+	Shell string `json:"shell" enum:"bash, sh, powershell, cmd"`
```

Comment:
Shouldn't it be `one of` instead of `enum`? https://pkg.go.dev/github.com/go-playground/validator/v10#hdr-One_Of